app/javascript/app/providers/data-explorer-provider/data-explorer-provider-actions.js

```diff
@@ -46,15 +44,20 @@ export const fetchDataExplorer = createThunkAction(
           section
         ]}${parsedQuery ? `?${parsedQuery}` : ''}`
       )
+        .then(response =>
```

Comment:
Not sure if this is happening because of this changes on the data fetching, but when I filter with the selectors and then change tab (or page) when coming back to the filtered tab, the filter is already actyve but the selectors are empty (and the filter is not set on the URL)

_in this example I've selected `Cambridge Architectural Research` visited another tab and went back to `Emission Pathways`_

![image](https://user-images.githubusercontent.com/6906348/42823336-967bec50-89dd-11e8-8068-ea6c82621e23.png)